contributors/design-proposals/storage/volume-topology-scheduling.md

```diff
@@ -727,18 +832,131 @@ cannot find a node for the pod, then it will encounter scheduling failure and
 initiate the rollback.
 
 ### Making dynamic provisioning topology aware
```

Comment:
I'm going to integrate this entire section into the static binding section

Reply:
Just pushed another update for this.  What I plan to continue working on is ironing out the dynamic provisioning interface, and still trying to reconcile all the new APIs for StorageClass.